File: python/retrieve.py
from pymongo import MongoClient
client=MongoClient('localhost',27017)
db=client.TYITDB259743
def read():
    try:
        Col=db.MyCol.find()
        print("\nAll data from database TYITDB259743:")
        for MyCol in Col:
            print(MyCol)
    except Exception:
        print(str(e))

read()

File: php/retrieve.php
<?php
$m = new MongoClient();
echo "Connection to database successfully"; 
$db = $m -> MYDB259743",
echo "\n Database selected successfully" ; 
$col = $db-> MyCol;
echo "\n Collection selected successfully"; 
$cursor=$col ->find();
foreach ($cursor as $doc)
{
echo "<br/>";
echo $doc["name"]."<br/>"; 
echo $doc["age"]."<br/>"; 
echo $doc["dept"]."<br/>"; 
echo $doc["rollno"]."<br/>";
}
?>